pkg/security/api/api.proto

```diff
@@ -1,15 +1,24 @@
 syntax = "proto3";
 
+option go_package = "pkg/security/api";
+
 package api;
 
-message GetParams{}
+message GetEventParams{}
```

Comment:
nit: `GetEvents`

Reply:
That is the parameter of the rpc method, I renamed it as now we have 2 methods with two kind of params